lib/widgets/action_sheet.dart

```diff
@@ -163,6 +163,58 @@ class ActionSheetCancelButton extends StatelessWidget {
   }
 }
 
+/// Show a sheet of actions you can take on a channel.
+void showChannelActionSheet(BuildContext context, {
+  required int streamId,
+}) {
```

Comment:
One useful change was merged after you started working on this PR 🙂: please see the top of `showTopicActionSheet` in `main`, and use `PageRoot.contextOf(context)` the way we do there.